ament_clang_format/ament_clang_format/configuration/.clang-format

```diff
@@ -16,4 +16,15 @@ ContinuationIndentWidth: 2
 DerivePointerAlignment: false
 PointerAlignment: Middle
 ReflowComments: false
+SpaceBeforeCtorInitializerColon: true
```

Comment:
This seems to be the default value for the Google style already. So it shouldn't need to be set.

Same for several other options.